packages/puppeteer-core/package.json

```diff
@@ -68,24 +67,24 @@
     "generate:sources": {
       "command": "tsx tools/generate_sources.ts",
       "files": [
-        "tools/generate_sources.ts",
-        "src/templates/**"
+        "../../versions.js",
+        "package.json",
```

Comment:
Wireit uses `package-lock.json` to generate Hash for builds, why is this needed here?